source/docs/hardware/sensors/analog-inputs-hardware.rst

```diff
@@ -18,27 +18,26 @@ Connecting to roboRIO analog input ports
 
 .. warning:: **Never** directly connect the power pin to the ground pin on any port on the roboRIO!  This will trigger protection features on the roboRIO and may result in unexpected behavior.
 
-|roboRIO Analog Inputs|
+.. image:: images/roborio-ports/roborio-aio.png
```

Comment:
This is missing the power/status/top-right lights.

Reply:
RSL distance spacing is also wrong.

Reply:
I updated all the roboRio illustrations in e9717f4. Please let me know if you want me to change anything else about them.